feature/container/containerz/tests/container_lifecycle/containerz_test.go

```diff
@@ -45,18 +56,35 @@ func TestDeployAndStartContainer(t *testing.T) {
 		}
 	}
 
-	ret, err := cli.StartContainer(ctx, "cntrsrv", "latest", "./cntrsrv", "test-instance", client.WithPorts([]string{"60061:60061"}))
+	ret, err := cli.StartContainer(ctx, "cntrsrv", "latest", "./cntrsrv", instanceName, client.WithPorts([]string{"60061:60061"}))
 	if err != nil {
 		t.Fatalf("unable to start container: %v", err)
 	}
+
+	// wait for cntr container to come up.
+	time.Sleep(5 * time.Second)
```

Comment:
I presume there's nothing fancier we can do to determine whether this container is alive? Is this the one that implements `ping` such that the service can be checked -- or should we consider this a wider integration test?